libraries/plugins/query_txid/include/graphene/query_txid/query_txid_plugin.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
New cpp and hpp files need the MIT license. You can add author/company name into it like:
https://github.com/bitshares/bitshares-core/blob/f1518aa5e4be2e8c80bfa48228e86460a559d641/libraries/plugins/custom_operations/custom_evaluators.cpp